consensus/istanbul/backend/pos.go

```diff
@@ -103,13 +103,15 @@ func (sb *Backend) distributeEpochRewards(header *types.Header, state *state.Sta
 		return err
 	}
 
-	// Validator rewards were paid in cUSD, convert that amount to cGLD and add it to the Reserve
-	totalValidatorRewardsConvertedToGold, err := currency.Convert(totalValidatorRewards, &stableTokenAddress, nil)
+	currencyManager := currency.NewManager(nil, nil)
```

Comment:
We should almost certainly pass in a header and state here (IIUC the parent's header and state), even though it was not passed in before. Currently this represents a race condition in consensus logic. (Style note, we might inline the creation of the currency manager to when it is called)

Reply:
correct... not only a race condition, it also implies that we use previous block exchange rates, not current block ones. I've already mentioned this on slack.. but changing this is a HF

Reply:
I'll leave as it is, since this is currently master branch behvaior, but i've added an entry here https://www.notion.so/clabsco/Remove-quirks-list-HardFork-Changes-7ee3a0b904ea464db6af45aba9f94a4a for us to remember changing it